HOTT-3798: Adds test and thursday workflow

willfish opened the pull request:
### What?

I have added/removed/altered:

- [x] Added test workflow
- [x] Added scheduled thursday email/rotation workflow

### Why?

I am doing this because:

- These are required in order to productionise the Electronic Tariff File (ETF)

### AC

- Files are being persisted to the trade-tariff-reporting bucket
- Emails are being sent to the test users
- Integrations are happening on changes to the ETF repository code
- The Thursday workflow runs at 0800

willfish committed:
HOTT-3798: Adds test workflow

willfish committed:
HOTT-3798: Adds debug mode integration

willfish committed:
HOTT-3798: Fixes broken sendgrid references

willfish committed:
HOTT-3798: Use the electronic-tariff-file context

willfish committed:
HOTT-3798: Debug

willfish committed:
HOTT-3798: Fixes need for CLI interactive feedback

willfish committed:
HOTT-3798: Try and be DRY

willfish committed:
HOTT-3798: Rotate electronic tariff files

willfish committed:
HOTT-3798: Annotate script

willfish committed:
HOTT-3798: Install the aws/cli

amberstarlight reviewed (approved):
Shellcheck was happy with this script, and so am I with the comments. LGTM.

willfish committed:
HOTT-3798: Rely on lexical comparison and refactor

- Adds explicit switches/full switches to make the script more readable
- Adds more annotations around jq and the awk filtering

willfish commented:
Okay, I've made some adjustments so that we're just relying on lexical comparisons of the different keys and I'd clearly been having a brain fart and was missing a who whole bunch of things to delete.


Just deleted in a local run:
```{
    "Deleted": [
        {
            "Key": "uk/reporting/2023/06/08/electronic_tariff_file/csv/hmrc-tariff-quotas-2023-06-08.zip"
        },
        {
            "Key": "uk/reporting/2023/06/15/electronic_tariff_file/csv/hmrc-tariff-commodities-2023-06-15.zip"
        },
        {
            "Key": "uk/reporting/2023/06/29/electronic_tariff_file/csv/hmrc-tariff-certificates-2023-06-29.7z"
        },
        {
            "Key": "uk/reporting/2023/06/29/electronic_tariff_file/csv/hmrc-tariff-commodities-2023-06-29.zip"
        },
        {
            "Key": "uk/reporting/2023/05/18/electronic_tariff_file/csv/hmrc-tariff-mfn-2023-05-18.zip"
        },
        {
            "Key": "uk/reporting/2023/06/29/electronic_tariff_file/csv/hmrc-tariff-supplementary_units-2023-06-29.zip"
        },
        {
            "Key": "uk/reporting/2023/06/29/electronic_tariff_file/delta/measures_2023-06-29.7z"
        },
        {
            "Key": "uk/reporting/2023/06/29/electronic_tariff_file/delta/commodities_2023-06-29.zip"
        },
        {
            "Key": "uk/reporting/2023/05/26/electronic_tariff_file/csv/hmrc-tariff-supplementary_units-2023-05-26.7z"
        },
        {
            "Key": "uk/reporting/2023/06/15/electronic_tariff_file/csv/hmrc-tariff-mfn-2023-06-15.7z"
        },
        {
            "Key": "uk/reporting/2023/05/18/electronic_tariff_file/csv/hmrc-tariff-footnotes-2023-05-18.7z"
        },
        {
            "Key": "uk/reporting/2023/06/22/electronic_tariff_file/csv/hmrc-tariff-mfn-2023-06-22.7z"
        },
        {
            "Key": "uk/reporting/2023/06/15/electronic_tariff_file/csv/hmrc-tariff-certificates-2023-06-15.zip"
        },
        {
            "Key": "uk/reporting/2023/06/08/electronic_tariff_file/csv/hmrc-tariff-measures-2023-06-08.zip"
        },
        {
            "Key": "uk/reporting/2023/06/22/electronic_tariff_file/csv/hmrc-tariff-measure_type-2023-06-22.zip"
        },
        {
            "Key": "uk/reporting/2023/06/22/electronic_tariff_file/csv/hmrc-tariff-supplementary_units-2023-06-22.zip"
        },
        {
            "Key": "uk/reporting/2023/06/08/electronic_tariff_file/csv/hmrc-tariff-commodities-2023-06-08.7z"
        },
        {
            "Key": "uk/reporting/2023/06/29/electronic_tariff_file/icl_vme/hmrc-tariff-ascii-2023-06-29.zip"
        },
        {
            "Key": "uk/reporting/2023/06/22/electronic_tariff_file/csv/hmrc-tariff-footnotes-2023-06-22.7z"
        },
        {
            "Key": "uk/reporting/2023/06/29/electronic_tariff_file/csv/hmrc-tariff-geography-2023-06-29.zip"
        },
        {
            "Key": "uk/reporting/2023/06/15/electronic_tariff_file/csv/hmrc-tariff-mfn-2023-06-15.zip"
        },
        {
            "Key": "uk/reporting/2023/05/25/electronic_tariff_file/csv/hmrc-tariff-mfn-2023-05-25.zip"
        },
        {
            "Key": "uk/reporting/2023/06/08/electronic_tariff_file/csv/hmrc-tariff-footnotes-2023-06-08.7z"
        },
        {
            "Key": "uk/reporting/2023/06/15/electronic_tariff_file/csv/hmrc-tariff-additional_code-2023-06-15.zip"
        },
        {
            "Key": "uk/reporting/2023/06/22/electronic_tariff_file/csv/hmrc-tariff-geography-2023-06-22.zip"
        },
        {
            "Key": "uk/reporting/2023/05/26/electronic_tariff_file/csv/hmrc-tariff-certificates-2023-05-26.7z"
        },
        {
            "Key": "uk/reporting/2023/05/18/electronic_tariff_file/csv/hmrc-tariff-supplementary_units-2023-05-18.zip"
        },
        {
            "Key": "uk/reporting/2023/05/26/electronic_tariff_file/csv/hmrc-tariff-supplementary_units-2023-05-26.zip"
        },
        {
            "Key": "uk/reporting/2023/05/18/electronic_tariff_file/icl_vme/hmrc-tariff-ascii-2023-05-18.7z"
        },
        {
            "Key": "uk/reporting/2023/06/15/electronic_tariff_file/icl_vme/hmrc-tariff-ascii-2023-06-15.7z"
        },
        {
            "Key": "uk/reporting/2023/05/26/electronic_tariff_file/delta/commodities_2023-05-26.zip"
        },
        {
            "Key": "uk/reporting/2023/05/26/electronic_tariff_file/icl_vme/hmrc-tariff-ascii-2023-05-26.7z"
        },
        {
            "Key": "uk/reporting/2023/06/08/electronic_tariff_file/csv/hmrc-tariff-supplementary_units-2023-06-08.7z"
        },
        {
            "Key": "uk/reporting/2023/06/15/electronic_tariff_file/csv/hmrc-tariff-quotas-2023-06-15.zip"
        },
        {
            "Key": "uk/reporting/2023/06/22/electronic_tariff_file/csv/hmrc-tariff-measures-2023-06-22.zip"
        },
        {
            "Key": "uk/reporting/2023/05/18/electronic_tariff_file/csv/hmrc-tariff-footnotes-2023-05-18.zip"
        },
        {
            "Key": "uk/reporting/2023/06/15/electronic_tariff_file/csv/hmrc-tariff-measure_type-2023-06-15.7z"
        },
        {
            "Key": "uk/reporting/2023/06/22/electronic_tariff_file/csv/hmrc-tariff-certificates-2023-06-22.7z"
        },
        {
            "Key": "uk/reporting/2023/05/18/electronic_tariff_file/icl_vme/hmrc-tariff-ascii-2023-05-18.zip"
        },
        {
            "Key": "uk/reporting/2023/06/22/electronic_tariff_file/csv/hmrc-tariff-commodities-2023-06-22.7z"
        },
        {
            "Key": "uk/reporting/2023/05/26/electronic_tariff_file/csv/hmrc-tariff-geography-2023-05-26.7z"
        },
        {
            "Key": "uk/reporting/2023/05/25/electronic_tariff_file/csv/hmrc-tariff-commodities-2023-05-25.zip"
        },
        {
            "Key": "uk/reporting/2023/06/29/electronic_tariff_file/delta/commodities_2023-06-29.7z"
        },
        {
            "Key": "uk/reporting/2023/06/08/electronic_tariff_file/csv/hmrc-tariff-mfn-2023-06-08.7z"
        },
        {
            "Key": "uk/reporting/2023/06/08/electronic_tariff_file/csv/hmrc-tariff-supplementary_units-2023-06-08.zip"
        },
        {
            "Key": "uk/reporting/2023/06/08/electronic_tariff_file/delta/commodities_2023-06-08.zip"
        },
        {
            "Key": "uk/reporting/2023/06/22/electronic_tariff_file/csv/hmrc-tariff-commodities-2023-06-22.zip"
        },
        {
            "Key": "uk/reporting/2023/06/29/electronic_tariff_file/csv/hmrc-tariff-certificates-2023-06-29.zip"
        },
        {
            "Key": "uk/reporting/2023/06/08/electronic_tariff_file/csv/hmrc-tariff-measure_type-2023-06-08.7z"
        },
        {
            "Key": "uk/reporting/2023/05/25/electronic_tariff_file/csv/hmrc-tariff-geography-2023-05-25.7z"
        },
        {
            "Key": "uk/reporting/2023/06/08/electronic_tariff_file/icl_vme/hmrc-tariff-ascii-2023-06-08.7z"
        },
        {
            "Key": "uk/reporting/2023/05/18/electronic_tariff_file/csv/hmrc-tariff-commodities-2023-05-18.zip"
        },
        {
            "Key": "uk/reporting/2023/05/18/electronic_tariff_file/csv/hmrc-tariff-supplementary_units-2023-05-18.7z"
        },
        {
            "Key": "uk/reporting/2023/06/15/electronic_tariff_file/csv/hmrc-tariff-supplementary_units-2023-06-15.zip"
        },
        {
            "Key": "uk/reporting/2023/05/25/electronic_tariff_file/csv/hmrc-tariff-geography-2023-05-25.zip"
        },
        {
            "Key": "uk/reporting/2023/05/18/electronic_tariff_file/csv/hmrc-tariff-quotas-2023-05-18.zip"
        },
        {
            "Key": "uk/reporting/2023/05/25/electronic_tariff_file/csv/hmrc-tariff-measure_type-2023-05-25.7z"
        },
        {
            "Key": "uk/reporting/2023/06/15/electronic_tariff_file/csv/hmrc-tariff-measures-2023-06-15.zip"
        },
        {
            "Key": "uk/reporting/2023/06/08/electronic_tariff_file/csv/hmrc-tariff-geography-2023-06-08.zip"
        },
        {
            "Key": "uk/reporting/2023/06/08/electronic_tariff_file/csv/hmrc-tariff-mfn-2023-06-08.zip"
        },
        {
            "Key": "uk/reporting/2023/05/18/electronic_tariff_file/csv/hmrc-tariff-measure_type-2023-05-18.7z"
        },
        {
            "Key": "uk/reporting/2023/05/26/electronic_tariff_file/csv/hmrc-tariff-measures-2023-05-26.zip"
        },
        {
            "Key": "uk/reporting/2023/06/22/electronic_tariff_file/csv/hmrc-tariff-certificates-2023-06-22.zip"
        },
        {
            "Key": "uk/reporting/2023/06/15/electronic_tariff_file/csv/hmrc-tariff-geography-2023-06-15.7z"
        },
        {
            "Key": "uk/reporting/2023/06/22/electronic_tariff_file/csv/hmrc-tariff-measure_type-2023-06-22.7z"
        },
        {
            "Key": "uk/reporting/2023/05/18/electronic_tariff_file/csv/hmrc-tariff-certificates-2023-05-18.zip"
        },
        {
            "Key": "uk/reporting/2023/05/26/electronic_tariff_file/delta/commodities_2023-05-26.7z"
        },
        {
            "Key": "uk/reporting/2023/06/15/electronic_tariff_file/delta/commodities_2023-06-15.7z"
        },
        {
            "Key": "uk/reporting/2023/06/29/electronic_tariff_file/csv/hmrc-tariff-footnotes-2023-06-29.7z"
        },
        {
            "Key": "uk/reporting/2023/05/18/electronic_tariff_file/csv/hmrc-tariff-measures-2023-05-18.7z"
        },
        {
            "Key": "uk/reporting/2023/05/18/electronic_tariff_file/csv/hmrc-tariff-quotas-2023-05-18.7z"
        },
        {
            "Key": "uk/reporting/2023/05/26/electronic_tariff_file/csv/hmrc-tariff-measure_type-2023-05-26.zip"
        },
        {
            "Key": "uk/reporting/2023/05/25/electronic_tariff_file/icl_vme/hmrc-tariff-ascii-2023-05-25.7z"
        },
        {
            "Key": "uk/reporting/2023/05/26/electronic_tariff_file/csv/hmrc-tariff-additional_code-2023-05-26.zip"
        },
        {
            "Key": "uk/reporting/2023/05/25/electronic_tariff_file/icl_vme/hmrc-tariff-ascii-2023-05-25.zip"
        },
        {
            "Key": "uk/reporting/2023/06/08/electronic_tariff_file/csv/hmrc-tariff-certificates-2023-06-08.zip"
        },
        {
            "Key": "uk/reporting/2023/05/25/electronic_tariff_file/csv/hmrc-tariff-footnotes-2023-05-25.7z"
        },
        {
            "Key": "uk/reporting/2023/05/26/electronic_tariff_file/csv/hmrc-tariff-measure_type-2023-05-26.7z"
        },
        {
            "Key": "uk/reporting/2023/06/22/electronic_tariff_file/csv/hmrc-tariff-mfn-2023-06-22.zip"
        },
        {
            "Key": "uk/reporting/2023/06/22/electronic_tariff_file/delta/measures_2023-06-22.7z"
        },
        {
            "Key": "uk/reporting/2023/05/25/electronic_tariff_file/csv/hmrc-tariff-certificates-2023-05-25.zip"
        },
        {
            "Key": "uk/reporting/2023/05/18/electronic_tariff_file/csv/hmrc-tariff-mfn-2023-05-18.7z"
        },
        {
            "Key": "uk/reporting/2023/05/25/electronic_tariff_file/csv/hmrc-tariff-measures-2023-05-25.zip"
        },
        {
            "Key": "uk/reporting/2023/06/29/electronic_tariff_file/csv/hmrc-tariff-supplementary_units-2023-06-29.7z"
        },
        {
            "Key": "uk/reporting/2023/05/25/electronic_tariff_file/csv/hmrc-tariff-quotas-2023-05-25.7z"
        },
        {
            "Key": "uk/reporting/2023/05/26/electronic_tariff_file/icl_vme/hmrc-tariff-ascii-2023-05-26.zip"
        },
        {
            "Key": "uk/reporting/2023/06/29/electronic_tariff_file/csv/hmrc-tariff-measure_type-2023-06-29.7z"
        },
        {
            "Key": "uk/reporting/2023/06/08/electronic_tariff_file/csv/hmrc-tariff-additional_code-2023-06-08.zip"
        },
        {
            "Key": "uk/reporting/2023/06/22/electronic_tariff_file/csv/hmrc-tariff-footnotes-2023-06-22.zip"
        },
        {
            "Key": "uk/reporting/2023/06/22/electronic_tariff_file/icl_vme/hmrc-tariff-ascii-2023-06-22.7z"
        },
        {
            "Key": "uk/reporting/2023/06/15/electronic_tariff_file/csv/hmrc-tariff-supplementary_units-2023-06-15.7z"
        },
        {
            "Key": "uk/reporting/2023/05/25/electronic_tariff_file/csv/hmrc-tariff-measure_type-2023-05-25.zip"
        },
        {
            "Key": "uk/reporting/2023/05/18/electronic_tariff_file/delta/commodities_2023-05-18.zip"
        },
        {
            "Key": "uk/reporting/2023/06/08/electronic_tariff_file/delta/measures_2023-06-08.zip"
        },
        {
            "Key": "uk/reporting/2023/05/26/electronic_tariff_file/delta/measures_2023-05-26.zip"
        },
        {
            "Key": "uk/reporting/2023/06/08/electronic_tariff_file/csv/hmrc-tariff-additional_code-2023-06-08.7z"
        },
        {
            "Key": "uk/reporting/2023/06/22/electronic_tariff_file/csv/hmrc-tariff-measures-2023-06-22.7z"
        },
        {
            "Key": "uk/reporting/2023/06/22/electronic_tariff_file/csv/hmrc-tariff-supplementary_units-2023-06-22.7z"
        },
        {
            "Key": "uk/reporting/2023/05/26/electronic_tariff_file/csv/hmrc-tariff-mfn-2023-05-26.zip"
        },
        {
            "Key": "uk/reporting/2023/05/26/electronic_tariff_file/csv/hmrc-tariff-mfn-2023-05-26.7z"
        },
        {
            "Key": "uk/reporting/2023/05/26/electronic_tariff_file/csv/hmrc-tariff-additional_code-2023-05-26.7z"
        },
        {
            "Key": "uk/reporting/2023/05/26/electronic_tariff_file/csv/hmrc-tariff-footnotes-2023-05-26.zip"
        },
        {
            "Key": "uk/reporting/2023/06/29/electronic_tariff_file/csv/hmrc-tariff-footnotes-2023-06-29.zip"
        },
        {
            "Key": "uk/reporting/2023/05/18/electronic_tariff_file/csv/hmrc-tariff-measure_type-2023-05-18.zip"
        },
        {
            "Key": "uk/reporting/2023/05/18/electronic_tariff_file/csv/hmrc-tariff-geography-2023-05-18.zip"
        },
        {
            "Key": "uk/reporting/2023/06/15/electronic_tariff_file/csv/hmrc-tariff-geography-2023-06-15.zip"
        },
        {
            "Key": "uk/reporting/2023/06/29/electronic_tariff_file/csv/hmrc-tariff-geography-2023-06-29.7z"
        },
        {
            "Key": "uk/reporting/2023/06/29/electronic_tariff_file/csv/hmrc-tariff-commodities-2023-06-29.7z"
        },
        {
            "Key": "uk/reporting/2023/05/26/electronic_tariff_file/csv/hmrc-tariff-footnotes-2023-05-26.7z"
        },
        {
            "Key": "uk/reporting/2023/06/08/electronic_tariff_file/csv/hmrc-tariff-quotas-2023-06-08.7z"
        },
        {
            "Key": "uk/reporting/2023/06/15/electronic_tariff_file/csv/hmrc-tariff-certificates-2023-06-15.7z"
        },
        {
            "Key": "uk/reporting/2023/06/15/electronic_tariff_file/delta/measures_2023-06-15.zip"
        },
        {
            "Key": "uk/reporting/2023/06/08/electronic_tariff_file/csv/hmrc-tariff-measure_type-2023-06-08.zip"
        },
        {
            "Key": "uk/reporting/2023/05/25/electronic_tariff_file/csv/hmrc-tariff-measures-2023-05-25.7z"
        },
        {
            "Key": "uk/reporting/2023/05/26/electronic_tariff_file/csv/hmrc-tariff-quotas-2023-05-26.7z"
        },
        {
            "Key": "uk/reporting/2023/06/08/electronic_tariff_file/csv/hmrc-tariff-commodities-2023-06-08.zip"
        },
        {
            "Key": "uk/reporting/2023/05/25/electronic_tariff_file/csv/hmrc-tariff-certificates-2023-05-25.7z"
        },
        {
            "Key": "uk/reporting/2023/06/29/electronic_tariff_file/csv/hmrc-tariff-additional_code-2023-06-29.zip"
        },
        {
            "Key": "uk/reporting/2023/06/15/electronic_tariff_file/delta/commodities_2023-06-15.zip"
        },
        {
            "Key": "uk/reporting/2023/06/08/electronic_tariff_file/csv/hmrc-tariff-geography-2023-06-08.7z"
        },
        {
            "Key": "uk/reporting/2023/06/29/electronic_tariff_file/csv/hmrc-tariff-mfn-2023-06-29.zip"
        },
        {
            "Key": "uk/reporting/2023/05/26/electronic_tariff_file/delta/measures_2023-05-26.7z"
        },
        {
            "Key": "uk/reporting/2023/05/25/electronic_tariff_file/csv/hmrc-tariff-supplementary_units-2023-05-25.7z"
        },
        {
            "Key": "uk/reporting/2023/05/25/electronic_tariff_file/delta/measures_2023-05-25.zip"
        },
        {
            "Key": "uk/reporting/2023/06/15/electronic_tariff_file/csv/hmrc-tariff-footnotes-2023-06-15.zip"
        },
        {
            "Key": "uk/reporting/2023/05/18/electronic_tariff_file/csv/hmrc-tariff-additional_code-2023-05-18.7z"
        },
        {
            "Key": "uk/reporting/2023/05/18/electronic_tariff_file/csv/hmrc-tariff-commodities-2023-05-18.7z"
        },
        {
            "Key": "uk/reporting/2023/05/25/electronic_tariff_file/csv/hmrc-tariff-commodities-2023-05-25.7z"
        },
        {
            "Key": "uk/reporting/2023/05/26/electronic_tariff_file/csv/hmrc-tariff-measures-2023-05-26.7z"
        },
        {
            "Key": "uk/reporting/2023/05/18/electronic_tariff_file/csv/hmrc-tariff-geography-2023-05-18.7z"
        },
        {
            "Key": "uk/reporting/2023/05/26/electronic_tariff_file/csv/hmrc-tariff-certificates-2023-05-26.zip"
        },
        {
            "Key": "uk/reporting/2023/06/29/electronic_tariff_file/icl_vme/hmrc-tariff-ascii-2023-06-29.7z"
        },
        {
            "Key": "uk/reporting/2023/05/25/electronic_tariff_file/csv/hmrc-tariff-footnotes-2023-05-25.zip"
        },
        {
            "Key": "uk/reporting/2023/06/29/electronic_tariff_file/csv/hmrc-tariff-measures-2023-06-29.zip"
        },
        {
            "Key": "uk/reporting/2023/05/18/electronic_tariff_file/csv/hmrc-tariff-measures-2023-05-18.zip"
        },
        {
            "Key": "uk/reporting/2023/06/29/electronic_tariff_file/csv/hmrc-tariff-measures-2023-06-29.7z"
        },
        {
            "Key": "uk/reporting/2023/06/22/electronic_tariff_file/delta/commodities_2023-06-22.zip"
        },
        {
            "Key": "uk/reporting/2023/06/29/electronic_tariff_file/csv/hmrc-tariff-measure_type-2023-06-29.zip"
        },
        {
            "Key": "uk/reporting/2023/05/25/electronic_tariff_file/delta/commodities_2023-05-25.zip"
        },
        {
            "Key": "uk/reporting/2023/05/25/electronic_tariff_file/csv/hmrc-tariff-quotas-2023-05-25.zip"
        },
        {
            "Key": "uk/reporting/2023/05/26/electronic_tariff_file/csv/hmrc-tariff-commodities-2023-05-26.zip"
        },
        {
            "Key": "uk/reporting/2023/06/29/electronic_tariff_file/csv/hmrc-tariff-quotas-2023-06-29.7z"
        },
        {
            "Key": "uk/reporting/2023/05/25/electronic_tariff_file/csv/hmrc-tariff-mfn-2023-05-25.7z"
        },
        {
            "Key": "uk/reporting/2023/06/22/electronic_tariff_file/csv/hmrc-tariff-geography-2023-06-22.7z"
        },
        {
            "Key": "uk/reporting/2023/06/22/electronic_tariff_file/csv/hmrc-tariff-quotas-2023-06-22.7z"
        },
        {
            "Key": "uk/reporting/2023/05/25/electronic_tariff_file/delta/measures_2023-05-25.7z"
        },
        {
            "Key": "uk/reporting/2023/06/15/electronic_tariff_file/csv/hmrc-tariff-measures-2023-06-15.7z"
        },
        {
            "Key": "uk/reporting/2023/06/29/electronic_tariff_file/csv/hmrc-tariff-mfn-2023-06-29.7z"
        },
        {
            "Key": "uk/reporting/2023/06/08/electronic_tariff_file/csv/hmrc-tariff-footnotes-2023-06-08.zip"
        },
        {
            "Key": "uk/reporting/2023/05/26/electronic_tariff_file/csv/hmrc-tariff-commodities-2023-05-26.7z"
        },
        {
            "Key": "uk/reporting/2023/06/08/electronic_tariff_file/csv/hmrc-tariff-certificates-2023-06-08.7z"
        },
        {
            "Key": "uk/reporting/2023/05/26/electronic_tariff_file/csv/hmrc-tariff-quotas-2023-05-26.zip"
        },
        {
            "Key": "uk/reporting/2023/05/18/electronic_tariff_file/delta/measures_2023-05-18.zip"
        },
        {
            "Key": "uk/reporting/2023/06/29/electronic_tariff_file/csv/hmrc-tariff-quotas-2023-06-29.zip"
        },
        {
            "Key": "uk/reporting/2023/05/18/electronic_tariff_file/delta/commodities_2023-05-18.7z"
        },
        {
            "Key": "uk/reporting/2023/05/18/electronic_tariff_file/delta/measures_2023-05-18.7z"
        },
        {
            "Key": "uk/reporting/2023/06/29/electronic_tariff_file/csv/hmrc-tariff-additional_code-2023-06-29.7z"
        },
        {
            "Key": "uk/reporting/2023/06/08/electronic_tariff_file/delta/commodities_2023-06-08.7z"
        },
        {
            "Key": "uk/reporting/2023/06/22/electronic_tariff_file/delta/commodities_2023-06-22.7z"
        },
        {
            "Key": "uk/reporting/2023/06/22/electronic_tariff_file/delta/measures_2023-06-22.zip"
        },
        {
            "Key": "uk/reporting/2023/06/08/electronic_tariff_file/icl_vme/hmrc-tariff-ascii-2023-06-08.zip"
        },
        {
            "Key": "uk/reporting/2023/06/15/electronic_tariff_file/csv/hmrc-tariff-additional_code-2023-06-15.7z"
        },
        {
            "Key": "uk/reporting/2023/06/08/electronic_tariff_file/csv/hmrc-tariff-measures-2023-06-08.7z"
        },
        {
            "Key": "uk/reporting/2023/06/15/electronic_tariff_file/csv/hmrc-tariff-footnotes-2023-06-15.7z"
        },
        {
            "Key": "uk/reporting/2023/06/15/electronic_tariff_file/csv/hmrc-tariff-measure_type-2023-06-15.zip"
        },
        {
            "Key": "uk/reporting/2023/06/15/electronic_tariff_file/delta/measures_2023-06-15.7z"
        },
        {
            "Key": "uk/reporting/2023/05/25/electronic_tariff_file/csv/hmrc-tariff-additional_code-2023-05-25.zip"
        },
        {
            "Key": "uk/reporting/2023/06/15/electronic_tariff_file/icl_vme/hmrc-tariff-ascii-2023-06-15.zip"
        },
        {
            "Key": "uk/reporting/2023/06/22/electronic_tariff_file/csv/hmrc-tariff-additional_code-2023-06-22.zip"
        },
        {
            "Key": "uk/reporting/2023/05/18/electronic_tariff_file/csv/hmrc-tariff-additional_code-2023-05-18.zip"
        },
        {
            "Key": "uk/reporting/2023/05/25/electronic_tariff_file/csv/hmrc-tariff-additional_code-2023-05-25.7z"
        },
        {
            "Key": "uk/reporting/2023/06/22/electronic_tariff_file/csv/hmrc-tariff-quotas-2023-06-22.zip"
        },
        {
            "Key": "uk/reporting/2023/06/29/electronic_tariff_file/delta/measures_2023-06-29.zip"
        },
        {
            "Key": "uk/reporting/2023/05/26/electronic_tariff_file/csv/hmrc-tariff-geography-2023-05-26.zip"
        },
        {
            "Key": "uk/reporting/2023/06/22/electronic_tariff_file/icl_vme/hmrc-tariff-ascii-2023-06-22.zip"
        },
        {
            "Key": "uk/reporting/2023/06/08/electronic_tariff_file/delta/measures_2023-06-08.7z"
        },
        {
            "Key": "uk/reporting/2023/05/18/electronic_tariff_file/csv/hmrc-tariff-certificates-2023-05-18.7z"
        },
        {
            "Key": "uk/reporting/2023/05/25/electronic_tariff_file/csv/hmrc-tariff-supplementary_units-2023-05-25.zip"
        },
        {
            "Key": "uk/reporting/2023/06/15/electronic_tariff_file/csv/hmrc-tariff-commodities-2023-06-15.7z"
        },
        {
            "Key": "uk/reporting/2023/06/15/electronic_tariff_file/csv/hmrc-tariff-quotas-2023-06-15.7z"
        },
        {
            "Key": "uk/reporting/2023/06/22/electronic_tariff_file/csv/hmrc-tariff-additional_code-2023-06-22.7z"
        },
        {
            "Key": "uk/reporting/2023/05/25/electronic_tariff_file/delta/commodities_2023-05-25.7z"
        }
    ]
}
```

And with a rerun

```
No files to delete
```
